### PR TITLE
Add --exclude=<name> argument to exclude plugins or themes from updating

### LINKED ISSUE #14: Add --exclude=<name> argument for excluding plugins and themes from update
If you want to update all plugins and themes except for a specific few, it would be helpful to have `--exclude=<name>` to exclude one or more from update.

### PR DESCRIPTION
**Pull request by Nikschavan**
Fixes #14

### CODE REVIEW

File: src/Theme_Command.php [hunk 1]
```diff
@@ -589,6 +589,9 @@ public function get( $args, $assoc_args ) {
 	 *
 	 * [--all]
 	 * : If set, all themes that have updates will be updated.
+	 * 
+	 * [--exclude=<name>]
+	 * : Comma separated list of plugin names that should be excluded from updating.
```
Comment (danielbachhuber) [inline src/Theme_Command.php]: This should be "theme names"

File: src/Plugin_Command.php [hunk 2]
```diff
@@ -498,6 +498,9 @@ protected function install_from_repo( $slug, $assoc_args ) {
 	 *
```
Comment (danielbachhuber) [inline src/Plugin_Command.php]: Can you include functional tests for using this argument with both themes and plugins?

Comment (Nikschavan) [inline src/Plugin_Command.php]: How are the plugin updates emulated in the test environment?

Comment (danielbachhuber) [inline src/Plugin_Command.php]: > How are the plugin updates emulated in the test environment?

Have you looked at the existing tests? How do you think they could be emulated?

**Comment by danielbachhuber** [comment]
Thanks for your work on this, @Nikschavan

**Pull request merged**